web/core/src/jsMain/kotlin/androidx/compose/web/css/CSS.kt

```diff
@@ -87,7 +87,7 @@ interface CSSNumberish {
 }
 
 fun CSSNumberish.asNumber() = this.asDynamic() as? Number
-fun CSSNumberish.asCSSNumericValue(): CSSNumericValue? = this.asDynamic() as? CSSNumericValueJS
+fun CSSNumberish.asCSSNumericValue(): CSSNumericValue? = this.asDynamic() as? CSSNumericValue
```

Comment:
I'm definitely asking this without having a broader context in mind, but I'm just curious if it can be not nullable?

Reply:
I really see no reason why it can not so let me check!